pkg/utils/time_utils_test.go

```diff
@@ -12,11 +12,11 @@ func TestClock(t *testing.T) {
 	t1 := time.Now()
 	t2 := clk.Now()
 
-	assert.True(t, t2.After(t1))
+	assert.True(t, t2.Compare(t1) >= 0)
```

Comment:
Please note another way to write it could have been simpler


```suggestion
	assert.False(t, t2.Before(t1))
```